Package.swift

```diff
@@ -263,7 +263,12 @@ if ProcessInfo.processInfo.environment["SWIFTPM_LLBUILD_FWK"] == nil {
 if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
     package.dependencies += [
         .package(url: "https://github.com/apple/swift-tools-support-core.git", .branch("master")),
-        .package(url: "https://github.com/apple/swift-argument-parser.git", .exact("0.3.0")),
+        // The 'swift-argument-parser' version declared here must match that
```

Comment:
Slight nitpick here — it doesn't have to exactly match, but just be compatible.  Since the dependency is "up to next minor", it means that all other dependencies have to resolve to a 0.3.x dependency.  In general SwiftPM will pick the highest version that satisfies all the constrains, so it's fine for one package to specify "at least 0.3.0" and another to say "at least 0.3.2" and another to say "at least 0.3", as long as the package has a single tag that satisfies all requirements. So I think the working here would be that the version requirements have to be compatible, but they don't need to strictly match.  It's only a problem if one of them starts to require "0.4.x", for example.